src/components/navigation/_navigation.scss

```diff
@@ -42,16 +41,16 @@
         }
 
         &-header {
-            font-size: 1rem;
-            margin: 0.75rem 0 0.6rem;
+            margin: 0.75rem 0 0.5rem;
             padding: 0 0 0 1rem;
         }
     }
 
     &__item {
         border-left: 4px solid transparent;
         display: block;
-        margin: 0 0 0.2rem;
+        margin: 0 0 0.25rem;
+        @extend .ons-u-mb-no;
```

Comment:
Here we are adding a margin bottom and then removing it. We might be able to remove both these lines